### DISCUSSION TITLE
search by trace id is not working in skywalking bundled zipkin UI in both versions 9.4.0 and 9.5.0

### DISCUSSION THREAD

**Discussion by p1742000** [apache/skywalking#10978] [General]
Hello @wu-sheng,
we have successfully started working with skywalking version 9.5.0. we found some issue. In zipkin UI tab, search by traceId is not working. if we give traceid in the search bar, it's showing all traces sometimes (based on limits) and sometimes showing wrong traces.

<img width="1787" alt="MicrosoftTeams-image (5)" src="https://github.com/apache/skywalking/assets/61838673/bc46625d-d878-4ace-b013-7d17bfea7d21">

<img width="1787" alt="MicrosoftTeams-image (4)" src="https://github.com/apache/skywalking/assets/61838673/8a89835b-bab9-4768-a175-0ee1eff0438d">

Are we planning ahead to fix this issue in the upcoming versions ?

**Comment by wu-sheng** [comment]
This is not a kind of my service. Why ping me?

**Comment by wankai123** [comment]
Haven't found this issue on the demo (http://demo.skywalking.apache.org). 
You can check and share some browser network logs if something wrong.
